wallet/src/actors/worker/handlers/create_vtt.rs

```diff
@@ -1,15 +1,20 @@
 use actix::prelude::*;
 
 use crate::{actors::worker, types};
-use witnet_data_structures::transaction::Transaction;
+use witnet_data_structures::{fee::AbsoluteFee, transaction::Transaction};
 
 pub struct CreateVtt {
     pub wallet: types::SessionWallet,
     pub params: types::VttParams,
 }
 
+pub struct CreateVttResponse {
+    pub fee: AbsoluteFee,
+    pub transaction: Transaction,
+}
+
 impl Message for CreateVtt {
-    type Result = worker::Result<Transaction>;
+    type Result = worker::Result<CreateVttResponse>;
```

Comment:
If we cannot change the wallet API because of exchanges, isn't this also a breaking change?

Reply:
I may be wrong, but I believe return types in the actors inside `wallet/src/actors/worker/handlers` can be freely changed because they are not part of the Wallet API. Those other actors actually managing the JSONRPC methods are in `wallet/src/actors/app/handlers`.

Thus the `CreateVttResponse` message that is returned back to the client remains unchanged:
![image](https://user-images.githubusercontent.com/1316439/194041697-60052293-0bb2-41ca-b510-3e506c4fd2a1.png)

Yes, there are two `CreateVttResponse` messages for different actors at different layers, and the same happens with many other messages. That may have led to this confusion.

Reply:
You may be right, it looks like the `create_vtt` route is handled by this impl:

```
impl Handler<CreateVttRequest> for app::App {
    type Result = app::ResponseActFuture<CreateVttResponse>;
```

And that result type is unchanged.